langchain-core/src/callbacks/base.ts

```diff
@@ -259,6 +260,19 @@ abstract class BaseCallbackHandlerMethodsClass {
     tags?: string[]
   ): // eslint-disable-next-line @typescript-eslint/no-explicit-any
   Promise<any> | any;
+
+  handleCustomEvent?(
+    eventName: string,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    payload: any,
```

Comment:
`payload` vs. `data`?

Reply:
These are just positional args, will show in tooltips but not affect anything else.

Reply:
Will update anyway